compose-pay-button/src/main/java/com/google/pay/button/PayButton.kt

```diff
@@ -55,35 +58,50 @@ fun PayButton(
     type: ButtonType = ButtonType.Buy,
     radius: Dp = 100.dp,
     enabled: Boolean = true,
+    onError: (Throwable) -> Unit = {},
+    fallbackUi: @Composable (() -> Unit)? = null,
 ) {
 
+    var showFallback by remember { mutableStateOf(false) }
+
     val radiusPixelValue = with(LocalDensity.current) { radius.toPx().toInt() }
 
-    AndroidView(
-        modifier = modifier,
-        factory = { context ->
-            GmsPayButton(context).apply {
-                this.initialize(
-                    ButtonOptions.newBuilder()
-                        .setButtonTheme(theme.value)
-                        .setButtonType(type.value)
-                        .setCornerRadius(radiusPixelValue)
-                        .setAllowedPaymentMethods(allowedPaymentMethods)
-                        .build()
-                )
-            }
-        },
-        update = { button ->
-            button.apply {
-                alpha = if (enabled) FULL_ALPHA else HALF_ALPHA
-                isEnabled = enabled
+    if (!showFallback) {
+        AndroidView(
+            modifier = modifier,
+            factory = { context ->
+                GmsPayButton(context).apply {
+                    kotlin.runCatching {
+                        this.initialize(
+                            ButtonOptions.newBuilder()
+                                .setButtonTheme(theme.value)
+                                .setButtonType(type.value)
+                                .setCornerRadius(radiusPixelValue)
+                                .setAllowedPaymentMethods(allowedPaymentMethods)
+                                .build()
+                        )
```

Comment:
This block can be broken by "Fatal Exception: java.lang.IllegalStateException" or other potential crashes requiring handling and a potential fallback.